apache-rat-tasks/src/test/resources/antunit/report-bad-configurations.xml

```diff
@@ -59,7 +58,7 @@
   </target>
 
   <target name="testNoStylesheet">
-    <rat:report format="styled">
+    <rat:report styleReport="styled">
```

Comment:
Does that mean we need more docs about how to use RAT within the Ant sphere? Are these changes similar to the unintendedly breaking changes that were introduced with 0.16.0?
https://creadur.apache.org/rat/apache-rat-tasks/index.html
https://creadur.apache.org/rat/apache-rat-tasks/report.html

Reply:
I guess we should a not about the deprecation and the new attribute in:
https://github.com/apache/creadur-rat/blob/master/apache-rat-tasks/src/site/apt/report.apt